### PR TITLE
CNV#34681: alert-related RNs

### PR DESCRIPTION
**Pull request by ousleyp**
Version(s): 4.16 (RNs); no cherrypick

Issue: [CNV-34681](https://issues.redhat.com//browse/CNV-34681)

Link to docs preview: 
- [notable technical changes](https://78147--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-16-release-notes.html#virt-4-16-notable-technical-changes)
- [deprecated features](https://78147--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-16-release-notes.html#virt-4-16-deprecated) (for deprecated runbooks)

QE review:
- [x] QE has approved this change.

Additional information: 
- Note for reviewers: I refer to the runbooks having been removed, but I'm actually removing them in a follow-up PR. 
- Two related RNs in one PR
- I was unsure if "notable technical changes" was the best place for the notice about the runbooks location, but I didn't think it fit cleanly in deprecated/removed.
- Fixing capitalization of the section header
- ~Temporarily fixing the version number attributes in this PR (until https://github.com/openshift/openshift-docs/pull/78146 is merged + I rebase)~

### CODE REVIEW

File: virt/release_notes/virt-4-16-release-notes.adoc [hunk 1]
```diff
@@ -126,6 +131,8 @@ Deprecated features are included in the current release and supported. However,
 //CNV-32032 Release note: DEPRECATED FEATURE (Windows 2012R2 templates deprecated)
 * Support for Windows Server 2012 R2 templates is deprecated.
 
+//CNV-34681: Deprecated alerts
+* The alerts `KubeVirtComponentExceedsRequestedMemory` and `KubeVirtComponentExceedsRequestedCPU` are deprecated. You can safely ignore or xref:../../observability/monitoring/managing-alerts.adoc#silencing-alerts_managing-alerts[silence] them.
```
Comment (sradco) [inline virt/release_notes/virt-4-16-release-notes.adoc:135]: ignore the -> silence them

Comment (ousleyp) [reply virt/release_notes/virt-4-16-release-notes.adoc:135]: Hi @sradco , do you mean I should change it to just say:
```suggestion
* The alerts `KubeVirtComponentExceedsRequestedMemory` and `KubeVirtComponentExceedsRequestedCPU` are deprecated. You can safely xref:../../observability/monitoring/managing-alerts.adoc#silencing-alerts_managing-alerts[silence] them.
```

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

**Comment by GroceryBoyJr** [comment]
@ousleyp I like the "silence" link. :+1:

**Review by Ahmad-Hafe — APPROVED** [review]

**Pull request merged**